docker/init.sh

```diff
@@ -14,7 +14,7 @@ else
     # Uncomment to start redis to test message passing
     # redis-server --daemonize yes
 
-    if [[ -n $NODEMON ]] && [[ $NODEMON == "true" ]]; then
+    if [[ $NODEMON == "true" ]]; then
```

Comment:
Why did you get rid of the bash variable existence check?

Reply:
Just removing redundancy since bash already evals `[[ $NODEMON == "true" ]]` to 1 if `NODEMON` is unset.